_posts/2024-12-19-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,345 @@
+---
+layout: post
+title: "Moving data between the JBOD disks using Cruise Control"
+date: 2024-12-19
+author: shubham_rawat
+---
+
+Apache Kafka is a platform which provides durability and fault tolerance by storing messages on disks and JBOD storage is one of the storage configuration types supported by Kafka.
+The JBOD data storage configuration allows Kafka brokers to make use of multiple disks.
+Using JBOD storage, you can increase the data storage capacity for Kafka nodes, which can further lead to performance improvements.
+In case you plan to remove a disk, and it contains some partition replicas, then you need to make sure that data is safely moved to some other disks.
+If the data is not removed from the disk, and it is removed then potential data loss can happen.
+Currently, moving data between the JBOD disks is done using the `kafka-reassign-partitions.sh` tool which is not very user-friendly, therefore in Strimzi 0.45.0 we are introducing the ability to move data between the JBOD disks using Cruise Control
+
+## Cruise Control to move data between JBOD disks
```

Comment:
If you move all your headers one level down, the webpage would look nicer ... i.e. `##` -> `###`, `###` -> `####` etc.